sdk/python/kfp/components/_python_op.py

```diff
@@ -146,10 +147,17 @@ def annotation_to_type_struct(annotation):
         if not annotation or annotation == inspect.Parameter.empty:
```

Comment:
I am not sure if this is within the scope of this PR but should there be type definitions for parameters?

Reply:
In this case, the python annotation can be any type.